vector-stores/spring-ai-milvus-store/src/main/java/org/springframework/ai/vectorstore/MilvusVectorStore.java

```diff
@@ -69,6 +57,12 @@
 import org.springframework.util.Assert;
 import org.springframework.util.StringUtils;
 
+import java.util.ArrayList;
```

Comment:
Import order needs to be fixed as per https://github.com/spring-projects/spring-ai/blob/cdb590338ce356c121d201c7991235abb191dbd8/src/checkstyle/checkstyle.xml#L110 Will fix when merging.